python/lsst/ts/scheduler/config_schema.py

```diff
@@ -116,440 +129,446 @@ class inside the module.
+                decription: Camera readout time (seconds).
+                default: 2.0
+              shutter_time:
+                type: number
+                description: Time it takes to open/close camera shutter (seconds).
+                default: 1.0
+              filter_change_time:
+                type: number
+                description: Time it takes to perform a filter change (seconds).
+                default: 120.0
+              filter_max_changes_burst_num:
+                type: number
+                description: Maximum number of filter changes in a night.
+                default: 1.
+              filter_max_changes_burst_time:
+                type: number
+                description: Minimum time (units=seconds) between filter changes in a night.
+                default: 0
+              filter_max_changes_avg_num:
+                type: number
+                description: Maximum average number of filter changes per year.
+                default: 3000
+              filter_max_changes_avg_time:
+                type: number
+                description: Maximum time (units=seconds) for the average number of filter changes.
+                default: 31557600.0
+              filter_mounted:
+                type: array
+                description: >-
+                  Initial state for the mounted filters. Empty positions must be
+                  filled with id="" no (filter).
+                default: [g, r, i, z, y]
+              filter_removable:
+                type: array
+                description: List of mounted filters that are removable for swapping
+                default: [y, z]
+              filter_unmounted:
+                type: array
+                description: List of unmounted but available filters to swap
+                default: [u]
+            additionalProperties: false
+          optics_loop_corr:
+            type: object
+            properties:
+              tel_optics_ol_slope:
+                type: number
+                description: >-
+                    Delay factor for Open Loop optics correction,  in units of seconds/(degrees in ALT slew)
+                default: 0.2857
+              tel_optics_cl_delay:
+                type: array
+                description: >-
+                    Table of delay factors for Closed Loop optics correction according to the ALT slew range.
+                default:   [0.0, 36.0]
+              tel_optics_cl_alt_limit:
+                type: array
+                description: Altitude ranges for optics closed loop correction.
+                default:   [0.0, 9.0, 90.0]
+            additionalProperties: false
+          slew:
+            type: object
+            properties:
+              prereq_telalt:
+                type: array
+                description: >-
+                  Which activity is required to complete before telescope motion in altitude can start?
+                default: []
```

Comment:
Down to line 456 there are many arrays of strings. Please describe what values they all can take.

Reply:
great idea! I will improve the schema to add items type.